jwql/jwql_monitors/monitor_filesystem.py

```diff
@@ -98,7 +91,7 @@ def files_per_filter():
 
             obs_table = [Observations.get_product_list(batch) for batch in batches]
             products = unique(vstack(obs_table), keys='productFilename')
-            filtered_products = Observations.filter_products(products, productType=["SCIENCE"], productSubGroupDescription=['UNCAL'], extension="fits")
+            Observations.filter_products(products, productType=["SCIENCE"], productSubGroupDescription=['UNCAL'], extension="fits")
```

Comment:
Does this statement have any effect?

Reply:
From where I'm sitting, it looks like a good majority of this function can be deleted. The object that gets returned is populated without any of the calls to filter products or deal with batch size etc.